index.html

```diff
@@ -35,6 +35,136 @@ <h1>I'm Yohannes</h1>
 
        </ul>
     
+    </section >
+    <section  class="work-section">
+        <h2 >My Recent Works</h2>
+        </hr>
+        <div class="MY-works">
+            <img class="placeholder" src="icons/Img Placeholder.png" alt="gymfit"/>
+            <div class="post stories">
+            <span class="mini-title"> Multi-Post Stories</span>
+            <p class='dummy-text'>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>  
+            <div class="css-container">
+
+                <div class="css">
+                    <span>css</span>
+                </div>
+                <div class="css">
+                    <span>html</span>
+                </div>
+                <div class="css">
+                    <span>bootstrap</span>
+                </div>
+                <div class="css">
+                    <span>Ruby</span>
+                </div>
```

Comment:
- [ ] [**OPTIONAL**] From the requirements of this project, these lists of project technologies are a list `ul` and each of them a `li`. Unordered list is good for presentation and readability of your HTML code, it also applies meaning to content.